sfcasts/turbo/stream-everything.md

```diff
@@ -96,7 +96,7 @@ In `symfony/ux-turbo` 2.1 and higher, this code has changed:
 
 ```php
 if (TurboBundle::STREAM_FORMAT === $request->getPreferredFormat()) {
-    $request->setFormat(TurboBundle::STREAM_FORMAT);
+    $request->setRequestFormat(TurboBundle::STREAM_FORMAT);
```

Comment:
Hm, I'm not sure about this change. I suppose the example was taken from the official docs: https://symfony.com/bundles/ux-turbo/current/index.html#forms - there we have `$request->setFormat(TurboBundle::STREAM_FORMAT);`. If it does not work - then it should be fixed upstream in the Symfony docs as well

Reply:
Its not working, there are several issues about it on symfony github, as well as pull requests.

https://github.com/symfony/ux/pull/297

```
/**
 * Associates a format with mime types.
 * /
public function setFormat(?string $format, $mimeTypes)

/**
 * Sets the request format.
*/
public function setRequestFormat(?string $format)

```

Reply:
Yeah, I see it, you're right! And it seems it was already fixed upstream in the docs I referenced above 👍